openerp-client/src/main/java/org/bahmni/openerp/web/client/strategy/implementation/OpenERPXMLClient.java

```diff
@@ -88,13 +71,19 @@ private void login() {
         }
     }
 
-    public Object delete(String resource, Vector params) {
-        return execute(resource, "unlink", params);
+    @Override
+    public Object execute(OpenERPRequest openERPRequest){
+        login();
+        String request = RequestBuilder.buildNewXMLRequest(openERPRequest, id, database, password);
+        String response = httpClient().post("http://" + host + ":" + port + XML_RPC_OBJECT_ENDPOINT, request);
```

Comment:
@rahu1ramesh is protocol always `http` ?

Reply:
Yes @sivareddyp, so far for the odoo10 calls it's always been http.